src/test/java/com/training/web/tests/rccl/GuestAccountTest.java

```diff
@@ -101,4 +106,48 @@ public void testCreateGuestAccountAndValidatePageElements() throws NullPointerEx
         softAssertions.assertThat(homePage.isPlanNewCruiseVisible()).as("Plan new cruises button not visible")
                 .isTrue();
     }
+
+    @Test
+    public void testAccountCreationGuiLoyaltyAddition() {
+
+        String lastName = "Poole";
+        String loyaltyId = "137529822";
+
+        CreateAccount createAccount = new LoginPage().createAccount();
```

Comment:
new LoginPage().createAccount().enterName(firstName, lastName)
                .enterDateOfBirth()
                .selectCountry()
                .enterSecurityQuestion()
                .enterIdPassword(email,password)
                .acceptTerms();
Attach all this here directly